gateway/Makefile

```
@@ -28,15 +28,19 @@ service.yaml: service.yaml.in
 
 deployment.yaml: deployment.yaml.in build
 	sed -e "s,@sha@,$$(git rev-parse --short=12 HEAD),g" \
-	  -e "s,@image@,$(IMAGE)," \
+	  -e "s,@image@,$(GATEWAY_IMAGE)," \
 	  < $< > $@
 
 build: Dockerfile $(NGINX_CONF)
-	docker build -t gateway .
+	-docker pull debian:9.5
```

under what conditions should this silently fail?